lib/external_logger/send_error.rb

```
@@ -29,11 +28,19 @@ def class
 
   class << self
     def call(error, params = {}, &block)
+      self::ADDITIONAL_LOGGERS.each do |name, active|
+        send("send_error_to_#{name}", error, params) if active
+      end
```

Should we raise an error here if someone adds a new logger to ADDITIONAL_LOGGERS but forgets to send the write method for this logger?

---

tests will not pass. We can add some check explicitly but I dont think it is needed